testing/src/test/java/io/stargate/it/http/RestApiTest.java

```diff
@@ -362,6 +362,45 @@ public void getRowWithClustering() throws IOException {
     assertThat(rowResponse.getRows().get(0).get("firstName")).isEqualTo("Jane");
   }
 
+  @Test
+  public void getRowWithMixedClustering() throws IOException {
```

Comment:
nit: I think mixed types are essential, so maybe `getRowWithMixedClusteringKeyTypes` ?

Reply:
As in renaming just renaming the method?